packages/android-client/FishjamClient/src/main/java/com/fishjamcloud/client/media/LocalVideoTrack.kt

```diff
@@ -18,7 +17,7 @@ class LocalVideoTrack(
   mediaTrack: org.webrtc.VideoTrack,
   endpointId: String,
   metadata: Metadata,
-  private val capturer: Capturer,
+  public val capturer: Capturer,
```

Comment:
are you sure you want this to be `public` so that user can see it and mess with it? 
maybe at least `internal`?

Reply:
If it would be internal i would not see it in react-native android code

Reply:
the general idea is that the rn-client is like android-client user
so if you need something private from android-client to implement something in rn then it's better to implement something in android-client (if possible)